test/server_test.py

```diff
@@ -200,19 +217,22 @@ def prepare_config(tmpdir, drop_privileges="true"):
         run_dir=os.path.join(tmpdir, "run"),
         log_dir=os.path.join(tmpdir, "log"),
         drop_priv=drop_privileges,
+        remote_port=testutil.random_tcp_port(),
```

Comment:
We should keep port=0 for all tests, using random_tcp_port() is needed only for starting a server with a ticket.

You can add a port=0 keyword arguemnt, and call it with a random_tcp_port() only in the specific test that needs a known port.

Another solution, test this feature in another test module starting the server in the same process, when we have access to the random port.

Testing here is problematic since this module uses the control socket for testing if the server is ready, and this does not work with initial ticket.

Testing the trivial code parsing the ticket argument is not very important, we can test that the Server class loads a ticket if ticket argument is specified.